main.py

```diff
@@ -596,7 +596,7 @@ async def scrape_channel_media(
         os.mkdir(attachment_directory_filepath)
 
     # Iterate through each message in the channel
-    async for message in channel.history(limit=500, oldest_first=True):
+    async for message in channel.history(limit=1000, oldest_first=True):
```

Comment:
Maybe we should make this a constant?

In constants section:
```Python
# Maximum number of messages to scan for media attachments
MESSAGE_SCAN_LIMIT = 1000
```
On this line:
```suggestion
    async for message in channel.history(limit=MESSAGE_SCAN_LIMIT, oldest_first=True):
```

Reply:
We have a comment in the constants section saying to see Discord docs for LIMIT variables, so if we do this we should either update that comment or change this variable name

Reply:
Good shout!